code-mirror.yml

```diff
@@ -9,6 +9,7 @@ trigger:
     - release_3.0
     - release_4.0_hotfix
     - feature/*
+    - aibhandari/*
```

Comment:
why do you need personal branches mirrored? If you need to test your branches in CI, you can always add the ADO mirror as a remote and manually push to it:

```
git remote add ado <ado_repo_uri>
git checkout <branch>
git push ado <branch>
```

Reply:
done